docker06.txt

```diff
@@ -0,0 +1,10 @@
+1.  Containerization's layering mechanism expedites development by allowing incremental changes within specific layers, 
+enhancing efficiency by reusing cached layers, reducing build times, and facilitating swift iterations, 
+fostering seamless collaboration and resource optimization among teams.
+
+2. i think that the first method is better, because it makes me write less and not repeating code. 
```

Comment:
it's not really about repeating code as much it is about creating new layers which are equal for more memory space